### PR TITLE
docs: updated pr template

### PR DESCRIPTION
**Pull request by emilyjablonski**
This PR addresses #(insert-number-here)

- [ ] Addresses the issue in full
- [ ] Addresses only certain aspects of the issue

## Description

Please include a summary of the change and which issue(s) is addressed.

## How Can This Be Tested/Reviewed?

Provide instructions so we can review, including any needed configuration, and the test cases that need to be QAd.

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [ ] Reviewed in a desktop view
- [ ] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [ ] Added tests covering the changes
- [ ] Made corresponding changes to the documentation

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

------------------------------

The above is the updated template. I removed some outdated information, and then generally tried to make it smaller, and to make the checklist more focused so it can be easier to complete.

### CODE REVIEW

**Review by ColinBuyck — APPROVED** [review]
Loving the shortened/more applicable checklist 🪤

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -1,52 +1,29 @@
...
-- [ ] I have assigned reviewers
-- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
-- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
-- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits
+- [ ] Added QA notes to the issue with applicable URLs
+- [ ] Reviewed in a desktop view
+- [ ] Reviewed in a mobile view
+- [ ] Reviewed considering accessibility
+- [ ] Added tests covering the changes
+- [ ] Made corresponding changes to the documentation
```
Comment (YazeedLoonat) [inline docs/pull_request_template.md:21]: sorry this is a late review here but can we also include: `I have run yarn generate:client` it catches me ALL the time forgetting to do it

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (emilyjablonski) [reply docs/pull_request_template.md:21]: Yes absolutely! Updated

**Review by YazeedLoonat — APPROVED** [review]
LGTM

**Pull request merged**